torchao/core/config.py

```diff
@@ -0,0 +1,10 @@
+import abc
+
+
+class AOBaseConfig(abc.ABC):
+    """
+    If a workflow config inherits from this then `quantize_` knows
+    how to a apply it to a model.
+    """
```

Comment:
should we add a paragraph here or under quantize_ about how this is related to `register_quantize_module_handler`, so users who wish to add their own configs know how to do it?